### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by danpalmer**
This PR adds a structure in which we can write tests, and a basic set of tests for the public API. It tests against Python 2.7 and Django 1.8 using Tox.

There are many more things that could be done here – Django 1.9 support, adding and testing migration support, Python 3 compatibility, etc, but these are saved for future PRs.

### CODE REVIEW

File: django_enumfield/enum.py [hunk 1]
```diff
@@ -35,6 +35,9 @@ def from_value(self, value):
             raise ValueError("%r is not a valid value for the enum" % value)
 
     def from_slug(self, slug):
+        if not isinstance(slug, str):
```
Comment (lamby) [inline django_enumfield/enum.py]: ``basestring``, not ``str`` in py2

**Comment by lamby** [comment]
Great stuff. If you could just fix that basestring thing I'll merge.

File: django_enumfield/enum.py (hunk 1 above)
Comment (danpalmer) [inline django_enumfield/enum.py]: 9e31cf1

**Comment by danpalmer** [comment]
Fixed. I'm going to try and submit a few other PRs soon so it might be worth adding Travis/Circle to the repo for testing now that there is something to run.

**Pull request merged**